diff --git a/lib/Varien/Data/Collection/Db.php b/lib/Varien/Data/Collection/Db.php
index a1debc5..f40f8e5 100644
--- a/lib/Varien/Data/Collection/Db.php
+++ b/lib/Varien/Data/Collection/Db.php
@@ -410,6 +410,19 @@ class Varien_Data_Collection_Db extends Varien_Data_Collection
      */
     protected function _translateCondition($field, $condition)
     {
+        // Temporary patch to determine whether there's something affected by
+        // APPSEC-1063 that we didn't catch. This should be rolled back after the logs have
+        // been reviewed
+        //
+        // NOTE: There will be an uncaught exception if this happens...
+        if (trim($field, 'A..Za..z0..9_') !== '') {
+            // Ruh-roh, we didn't catch this one..
+            Mage::log($field, null, 'appsec-1063.log', true);
+            Mage::log(json_encode($condition), null, 'appsec-1063.log', true);
+            Mage::log(Varien_Debug::backtrace(true, false), null, 'appsec-1063.log', true);
+            Mage::log('===============', null, 'appsec-1063.log', true);
+        }
+
         $mappedField = $this->_getMappedField($field);
 
         $quotedField = $mappedField;
